corehq/apps/export/models/new.py

```diff
@@ -827,9 +832,16 @@ class Meta(object):
     @classmethod
     def wrap(cls, data):
         from corehq.apps.export.views.utils import clean_odata_columns
+        # This is a temporary solution to make sure geojson exports have split_multiselects set to false,
+        # otherwise it won't work. Ticket SC-3569 is for making geojson form exports compatible with
+        # the split_multiselects option.
+        if data.get('export_format', '') == 'geojson':
+            data['split_multiselects'] = False
```

Comment:
Just so I know, what does `split_multiselects` do? And how that breaks geojson export?

Reply:
The `split_multiselects` option takes the GPS coordinate columns (it works on other columns as well) and splits it into multiple columns, one for longitude, latitude and accuracy, instead of having one column with the GPS coordinate. This essentially means that it's not a straight forward process to determine which column to use when we're trying to figure out which columns the `selected_geo_property` value speaks to when exporting, since there's now multiple columns which need to be aggregated to obtain the `selected_geo_property` column's value.

All in all I think this problem needs to be thought through a bit, hence why I'm not doing it in this PR.

Reply:
Cool. Totally agreed to taking it separately.